File: python/exercício_29.py
print()
print("=-"*60)
print()
print("--------------Desconto de salario------------------")
print()
print("=-"*60)
print() 
salario=float(input("Digite o seu salario: "))
print()
print("=-"*60)
print()
desconto = salario * 0.89
a = salario - desconto
if a <= 334.29:
    print("Seu salario é {} e com desconto ficou {}".format(salario,desconto))
    print()
    print("=-"*60)
    print()
else: 
    b= salario - 334.29
    print("O salarios {} ultrapassou o limite de desconto de R$:334.29 então o novo dalario ficou: {}".format(salario, b))
    print()
    print("=-"*60)
    print()

File: java/exercício_29.java
import java.util.Scanner;

public class exercício_29 {
    public static void main(String[] args) {
        try (Scanner resul = new Scanner(System.in)){;

        System.out.printf("Digite o seu salario:");
        double salario = resul.nextDouble();

        double desconto = salario * 0.89;
        double total = salario - desconto;

        if (total <= 334.29) {
            System.out.printf("Seu salario é" + salario + "e com desconto ficou:" + desconto);
        } else {
            double novosalario = salario - 334.29;
            System.out.printf("O salario" + salario
                    + "ultrapassou o limite de desconto de R$:334.29 então o novo dalario ficou:" + novosalario);
        }
    }
    }
}